mlflow_databricks_artifacts/utils/string_utils.py

```diff
@@ -0,0 +1,12 @@
+def strip_prefix(original, prefix):
+    if original.startswith(prefix):
+        return original[len(prefix) :]
+    return original
+
+
+def strip_suffix(original, suffix):
+    if original.endswith(suffix) and suffix != "":
+        return original[: -len(suffix)]
+    return original
+
```

Comment:
I have yet to lint / reformat this package, which I'll make sure to do. Empty newlines here are a good example.